app/documentation/login.md

```diff
@@ -0,0 +1,25 @@
+# Emergency suppport Technical Documentation
+
+Emergency suppport feature documentation:
+
+- [ ] User clicks Sign in Button
+- [ ] User inputs user Name -> could be email or username itself
+- [ ] User inputs Password
+- [ ] User clicks Sign in
+- [ ] App sends data entered to backend to validate
+- [ ] Backend sends back login token
+- [ ] App saves login token in async storage
```

Comment:
@jyaneshseeam @ja-chisholm @DemiAdisa 
Please do not use checklist other than issue. Use order or unordered list instead for PR and Documentation

- Item 1
- Item 2
- Item 3

or 
1. Item A
2. Item B
3. Item C

Checkbox is used to track to do list

Reply:
@jyaneshseeam any progress?

Reply:
> @jyaneshseeam any progress?

just pushed new one, please approve

Reply:
@jyaneshseeam lint errors are there

Reply:
@sontushkpaul  I just edited the part causing the linting error in front end should be good now